src/api/base.ts

```diff
@@ -257,6 +257,25 @@ export class BaseAPI {
         }
     }
 
+    async updateCookies(identity: Identity)  {
```

Comment:
Please try to remove `GCLB` or `overleaf_session2` keywords in this function. I hope this function is more general.